### ISSUE TITLE
Expose a delete() function

### ISSUE THREAD

**Issue by BrunoKrugel** [gennaro-tedesco/nvim-possession#22]
Would be a nice feature to expose the `delete()` function, so we can delete the current session (like the update or save), without having to open the list and then delete manually the session.

**Comment by gennaro-tedesco** [comment]
It can be done: could you specify how you would like the function to work? If I understand correctly, it should simply check if you are in a loaded session and if so, delete it from disk?

**Comment by BrunoKrugel** [comment]
Exactly, so we can have, for example:

```
{
    "gennaro-tedesco/nvim-possession",
    dependencies = {
        "ibhagwan/fzf-lua",
    },
    config = true,
    init = function()
        local possession = require("nvim-possession")
        vim.keymap.set("n", "<leader>sl", function()
            possession.list()
        end)
        vim.keymap.set("n", "<leader>sn", function()
            possession.new()
        end)
        vim.keymap.set("n", "<leader>su", function()
            possession.update()
        end)
        vim.keymap.set("n", "<leader>sd", function()
            possession.delete()
        end)
    end,
}
```

**Comment by gennaro-tedesco** [comment]
I have created a [draft PR](https://github.com/gennaro-tedesco/nvim-possession/pull/23) to address this issue.

Could you please help me test if this is what you wished? You can install the branch `delete_local_session` (via your package manager of choice) and `<leader>sd` is the suggested mapping to be set (as you mention above).

**Comment by BrunoKrugel** [comment]
it works!

**Comment by gennaro-tedesco** [comment]
I will have another go at the code to see if I can polish it a little and test it again: will merge in the next days (I may be busy at work so bear with me, but I don't forget :p)

**Comment by gennaro-tedesco** [comment]
This change has been included in the latest release [v0.0.9](https://github.com/gennaro-tedesco/nvim-possession/releases/tag/v0.0.9). Just update your plugin via your package manager of choice and you're good to go :)
